rust/core-lib/src/event_context.rs

```diff
@@ -147,7 +147,7 @@ impl<'a> EventContext<'a> {
     pub(crate) fn do_plugin_cmd(&mut self, plugin: PluginId,
                                  cmd: PluginNotification) {
         use self::PluginNotification::*;
-
+        let viewID = self.view.borrow().view_id;
```

Comment:
Rust convention is to give identifiers `snake_case` names. (You should get a warning about this when you compile; in general warnings should be addressed unless there's a good reason.)